skopt/trees.py

```diff
@@ -0,0 +1,127 @@
+"""Tree based minimization algorithms"""
+
+import numpy as np
+
+from scipy import stats
+from scipy.optimize import fmin_l_bfgs_b
+from scipy.optimize import OptimizeResult
+
+from sklearn.base import clone
+from sklearn.utils import check_random_state
+
+from .gbt import GradientBoostingQuantileRegressor
+from .utils import extract_bounds
+
+
+def _expected_improvement(x, surrogate, best_y):
+    """Evaluate expected improvement for `surrogate` model at `x`"""
+    x = x.reshape(1, -1)
+    low, mu, high = surrogate.predict(x)
+    std = (high - low) / 2.
```

Comment:
We should mention that this an approximation to the standard deviation and need not be the exact standard deviation.